modules/virt-viewing-automatically-created-storage-profiles.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * virt/storage/virt-configuring-storage-profile.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-viewing-automatically-created-storage-profiles_{context}"]
+= Viewing automatically created storage profiles
+
+The system creates storage profiles for each storage class automatically.
+
+. To view the list of storage profiles, run the following command:
++
+[source,terminal]
+----
+oc get storageprofile
+----
+
+. To fetch the details of a particular storage profile, run the following command:
++
+[source,terminal]
+----
+oc describe storageprofile <name>
+----
+
+.Example storage profile details
+[source,yaml]
+----
+$ oc describe  storageprofile ocs-storagecluster-ceph-rbd-virtualization
```

Comment:
This seems repetitive with the above command and shouldn't be included in the YAML output.